### PR TITLE
Feature/easy admin

### PR DESCRIPTION
**Pull request by manolis59**
Découverte de git pr work in progress

### CODE REVIEW

File: src/Controller/Admin/BookCrudController.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+
+class BookCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Book::class;
+    }
+
+    /*
```
Comment (dunglas) [inline src/Controller/Admin/BookCrudController.php:15]: You can delete this code as it is unused.

File: src/Controller/Admin/BorrowCrudController.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+
+class BorrowCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Borrow::class;
+    }
+
+    /*
```
Comment (dunglas) [inline src/Controller/Admin/BorrowCrudController.php:15]: Same here.

File: src/Controller/Admin/DashboardController.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+use Symfony\Component\Routing\Annotation\Route;
+
+class DashboardController extends AbstractDashboardController
+{
+    #[Route('/admin', name: 'admin')]
+    public function index(): Response
+    {
+        return $this->render('admin/index.html.twig');
+
+        // Option 1. You can make your dashboard redirect to some common page of your backend
```
Comment (dunglas) [inline src/Controller/Admin/DashboardController.php:23]: Same here, remove the useless auto-generated code everywhere.

File: src/Controller/Admin/DashboardController.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        yield MenuItem::linkToDashboard('Dashboard', 'fa fa-home');
+        yield MenuItem::linkToCrud('Users', 'fas fa-users', User::class);
+        yield MenuItem::linkToCrud('Books', 'fas fa-book', Book::class);
+        yield MenuItem::linkToCrud('Physical Books', 'fas fa-layer-group', PhysicalBook::class);
+        yield MenuItem::linkToCrud('Reviews', 'fas fa-file-pdf', Review::class);
+        yield MenuItem::linkToCrud('Borrow', 'fas fa-calendar-days', Borrow::class);
+        
+        
+       
+        // yield MenuItem::linkToCrud('The Label', 'fas fa-list', EntityClass::class);
```
Comment (dunglas) [inline src/Controller/Admin/DashboardController.php:57]: To delete

File: src/Controller/Admin/PhysicalBookCrudController.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+
+class PhysicalBookCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return PhysicalBook::class;
+    }
+
+    /*
```
Comment (dunglas) [inline src/Controller/Admin/PhysicalBookCrudController.php:15]: To delete

File: src/Controller/Admin/ReviewCrudController.php [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+
+class ReviewCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Review::class;
+    }
+
+    /*
```
Comment (dunglas) [inline src/Controller/Admin/ReviewCrudController.php:15]: To delete

**Review by dunglas — CHANGES_REQUESTED** [review]

**Review by Margauxfeslard — CHANGES_REQUESTED** [review]
Can you delete this file please : templates/test2.html.twig ?

**Review by Margauxfeslard — APPROVED** [review]

File: src/Controller/Admin/UserCrudController.php [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+    public function configureFields(string $pageName): iterable
+    {
+        
+        yield TextField::new('id')
+                ->onlyOnIndex();
+        yield TextField::new('fullName')
+                ->hideOnForm();
+        yield EmailField::new('email');
+
```
Comment (dunglas) [inline src/Controller/Admin/UserCrudController.php:25]:
```suggestion
```
Blank line to remove (use PHP CS Fixer)

File: src/Controller/Admin/DashboardController.php [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+    public function configureMenuItems(): iterable
+    {
+        yield MenuItem::linkToDashboard('Dashboard', 'fa fa-home');
+        yield MenuItem::linkToCrud('Users', 'fas fa-users', User::class);
+        yield MenuItem::linkToCrud('Books', 'fas fa-book', Book::class);
+        yield MenuItem::linkToCrud('Physical Books', 'fas fa-layer-group', PhysicalBook::class);
+        yield MenuItem::linkToCrud('Reviews', 'fas fa-file-pdf', Review::class);
+        yield MenuItem::linkToCrud('Borrow', 'fas fa-calendar-days', Borrow::class);
+
```
Comment (dunglas) [inline src/Controller/Admin/DashboardController.php:38]:
```suggestion   
```

Blank line to remove (use PHP CS Fixer)

**Review by dunglas — APPROVED** [review]

**Pull request merged**